test/AddressSpaceCast/issue-1350.ll

```diff
@@ -0,0 +1,23 @@
+; RUN: clspv-opt %s -o %t.ll --passes=lower-addrspacecast
+; RUN: FileCheck %s < %t.ll
+
+; CHECK: declare void @_Z21llvm.memcpy.p0.p4.i32PPU3AS1memcpy.p0.p4.i32(ptr noalias nocapture writeonly, ptr addrspace(1) noalias nocapture readonly, i32, i1) #0
```

Comment:
Is `clspv` really able to deal with such a function to emit anything valid?
Maybe because it contains `llvm.memcpy` it works?

Reply:
Yeah, the replace intrinsics pass seems prepared for this, as it looks at the function name containing "llvm.memcpy".